.buildkite/scripts/e2e/bootstrap.py

```diff
@@ -0,0 +1,177 @@
+        if result.returncode != 0:
+            raise Exception("Could not make `elastic-package` global.")
+
+    def __clone_integrations_repo(self):
+        result = subprocess.run(["git", "clone", "https://github.com/elastic/integrations.git"])
+        if result.returncode != 0:
+            raise Exception(f"Error occurred while cloning an integrations repo. Check logs for details.")
+
+    def __get_profile_path(self):
+        return os.path.join(Util.get_home_path(), ".elastic-package/profiles/e2e")
+
+    def __create_config_file(self, sample_config_file, config_file):
+        with open(sample_config_file, "r") as infile, open(config_file, "w") as outfile:
+            for line in infile:
+                if "stack.logstash_enabled: true" in line:
+                    # Logstash is disabled by default, remove the comment
+                    line = line.lstrip('#').lstrip()
+                outfile.write(line)
+
+    def __setup_elastic_package_profile(self):
+        result = subprocess.run(["elastic-package", "profiles", "create", "e2e"])
+        if result.returncode != 0:
+            raise Exception(f"Error occurred while creating a profile. Check logs for details.")
+
+        print("`elastic-package` e2e profile created.")
+
+        # elastic-package creates a profile under home directory
+        config_example_file = os.path.join(self.__get_profile_path(), "config.yml.example")
+        config_file = os.path.join(self.__get_profile_path(), "config.yml")
+        self.__create_config_file(config_example_file, config_file)
+        subprocess.run(["elastic-package", "profiles", "use", "e2e"])
+
+    def __install_plugin(self):
+        with open("VERSION", "r") as f:
+            version = f.read()
+
+        plugin_name = f"logstash-filter-elastic_integration-{version.strip()}-java.gem"
+        container = Util.get_logstash_container()
+
+        tar_data = io.BytesIO()
+        with tarfile.open(fileobj=tar_data, mode='w') as tar:
+            tar.add(plugin_name, arcname=f"/usr/share/logstash/{plugin_name}")
+
+        tar_data.seek(0)
+        container.put_archive('/', tar_data.getvalue())
+
+        print("Installing logstash-filter-elastic_integration plugin...")
+        command = f"bin/logstash-plugin install {plugin_name}"
+        exec_result = container.exec_run(command)
+        if exec_result.exit_code != 0:
+            raise Exception("Error occurred installing plugin in Logstash container, "
+                            f"output: {exec_result.output.decode('utf-8')}.")
+        print("Plugin installed successfully.")
+
+    def __reload_container(self):
+        result = subprocess.run(["docker", "restart", f"{self.LOGSTASH_CONTAINER_NAME}"])
+        print(f"Logstash container restart result: {result}")
+        if result.returncode != 0:
+            raise Exception(f"Error occurred while reloading Logstash container, see logs for details.")
+
+    def __update_pipeline_config(self):
```

Comment:
review note: this requires Logstash run with `--config.reload.automatic` and `elastic-package` config is read-only now, will change.

Reply:
The relative: https://github.com/elastic/elastic-package/pull/1668